Formula/mysql@5.7.rb

```diff
@@ -44,6 +45,9 @@ def install
       # against `_ZN17Gcs_debug_options12m_debug_noneB5cxx11E' can not be used when making
       # a shared object; recompile with -fPIC
       ENV.append_to_cflags "-fPIC"
+      # Work around failure from GCC 10+ using default of `-fno-common
+      # innodb_engine.c.o:(.bss.ib_cb_cfg_trx_level+0x0): multiple definition of `ib_cb_cfg_trx_level';
+      ENV.append_to_cflags "-fcommon"
```

Comment:
When can this be removed?

Reply:
Given Oracle's own OS still carry around a GCC 10 patch for MySQL 5.7, I wouldn't be surprised if it was never.

MySQL 5.7 is EOL in October 2023.

Reply:
Should probably add a deprecation then. And a reference to the upstream patch